Bottom: 6f2a456960849d37dc43a77ea476a67b658609e9
Top:    8098571666685aa6a8d637532398e0f42a184f64
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:12:50 +0100

Refresh of hid-gtk-better-encapsulate-som

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 8c7a713..da96f57 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -311,8 +311,6 @@ ghid_layer_selector_init (GHidLayerSelector *ls)
                                                        "font", FONT_COL,
                                                        NULL);
 
-  ls = g_object_new (GHID_LAYER_SELECTOR_TYPE, NULL);
-
   /* action index, active, color, text, font, is_separator */
   ls->list_store = gtk_list_store_new (N_COLS, G_TYPE_POINTER, G_TYPE_INT,
                                        G_TYPE_BOOLEAN, G_TYPE_STRING,
